### PR TITLE
[IMP] upgrade: overhaul upgrade doc

### PR DESCRIPTION
**Pull request by nmarotte**
Overhauling of the documentation for Upgrade as part of the need for Partner documentation for upgrade (task 3171474 )

This keep most of the information intact, merging the FAQ into the documentation itself, improves the structure (one page), remove redundancies, ...

It is part of a 2-phases project where the second phase would be to add information about upgrade for developers (currently sitting in developer/reference/upgrade.rst but that will be moved to developer/howtos.html) to contain informations about migration scripts, upgrading custom code, etc ...

Related PR : #5774 #5953

### CODE REVIEW

**Comment by nmarotte** [comment]
Hello @xpl-odoo @AntoineVDV , this is the final PR for the first phase of the overhaul of the upgrade documentation. 

@xpl-odoo Around 95% of the changes were already reviewed by you, and the rest of the changes can be viewed here  (/!\ Only the administration/maintain/upgrade.rst is published, everything in developer/upgrade.rst is postponed, so no need to review it, as it is not part of this PR ) : https://github.com/odoo/documentation/pull/5953/files/89ad30ed5d3db3b5d9b3aebd7aca03e29e7d6a31..7233cb7a351d604345942331ec3d0264cad72cd4# (The SLA is simply copy pasted from what already existed before)

The other 5% are the various I changes I did on this PR, one to fix this 
image

Another one to remove the note about "valid" subscriptions (https://github.com/odoo/documentation/pull/5953/files#diff-eb11d05d17bcc7b76a9453b1cdca38bc15380c0e3fbda7d6f2e68aa63f521dffR128)

and the last one to remove a trailing whitespace

Don't hesitate to let me know if you want to re-review this one last time together, or if it looks fine.

@AntoineVDV I'll be waiting for your explicit green light for merging this PR. If accepted before the OXP, I will merge it on Wednesday during the OXP, probably during the Keynote, if you think that is a good idea (I'd also like your feedback about when to publish it @augustoPerez and @odoo-pda , don't hesitate to let me know if you want it to be published at a specific time and date)

Thanks everyone for your help :)

File: content/administration/upgrade.rst [hunk 1]
```diff
@@ -1,226 +1,371 @@
...
+
+   It is strongly recommended to manually :ref:`test the upgrade first <upgrade/test_your_db>`.
+   Clicking :guilabel:`I want to test first` redirects to `the database manager
+   <https://www.odoo.com/my/databases/>`_, where it is possible to request an upgraded test database
+   and check it for any discrepancies.
 
-   upgrade/odoo_online
-   upgrade/odoo_sh
-   upgrade/on_premise
-   upgrade/faq
```
Comment (AntoineVDV) [inline content/administration/upgrade.rst:18]: These pages should each be replaced by a [redirect rule](https://github.com/odoo/documentation/blob/16.0/redirects/MANUAL.md).

**Review by AntoineVDV — COMMENTED** [review]
Always happy to see the content tabs used in the doc! :tada: 

LTGM from a tech POV, except for the redirect rules (see comment below).

> If accepted before the OXP, I will merge it on Wednesday during the OXP, probably during the Keynote, if you think that is a good idea

I recommend you rather merge that ASAP today because you still need to forward-port it to 17.0, and I will create and merge the "publish 17.0" PR today, as the doc is built overnight. i.e., merge it the day before you want it published.

Also, is there a particular reason to merge this PR in 16.0 and not 15.0 (or even 14.0)? If not, you should create a new backport PR to put this current PR's content as-is in earlier versions so that the instructions are version-independent.

@robodoo delegate+

**Comment by nmarotte** [comment]
@AntoineVDV There is no particular reason for it to be pushed to 16.0 should I therefore retarget to 14.0 and pray for the least amount of conflict ? :D

I will work on the redirect now :D

File: redirects/14.0.txt [hunk 2]
```diff
@@ -8,7 +8,11 @@ administration/maintain/online.rst administration/maintain/odoo_online.rst
 
 # administration/upgrade
 
-administration/upgrade/online_hosting.rst administration/upgrade/odoo_online.rst
+administration/upgrade/online_hosting.rst administration/upgrade/upgrade.rst
+administration/upgrade/faq.rst administration/upgrade/upgrade.rst
+administration/upgrade/odoo_online.rst administration/upgrade/upgrade.rst
+administration/upgrade/odoo_sh.rst administration/upgrade/upgrade.rst
+administration/upgrade/on_premise.rst administration/upgrade/upgrade.rst
```
Comment (AntoineVDV) [inline redirects/14.0.txt:15]:
```suggestion
administration/upgrade/online_hosting.rst administration/upgrade.rst
administration/upgrade/faq.rst administration/upgrade.rst
administration/upgrade/odoo_online.rst administration/upgrade.rst
administration/upgrade/odoo_sh.rst administration/upgrade.rst
administration/upgrade/on_premise.rst administration/upgrade.rst
```

Comment (nmarotte) [reply redirects/14.0.txt:15]: Whoops, nice catch haha

**Comment by nmarotte** [comment]
robodoo r+

**Pull request closed without merging**